crawlerr/models/rag_models.py

```
@@ -0,0 +1,137 @@
+class SearchMatch(BaseModel):
+    model_config = ConfigDict()
+
+    """A search result match with similarity score."""
+    document: DocumentChunk
+    score: float = Field(ge=0.0, le=1.0)
+    relevance: str = Field(default="medium")  # low, medium, high
+    highlighted_content: Optional[str] = None
+    
+    @field_validator("relevance", mode="before")
+    @classmethod
+    def calculate_relevance(cls, v, info):
```

Ruff (0.12.2)

Missing return type annotation for classmethod `calculate_relevance`

(ANN206)